content/en/docs/tasks/traffic-management/egress/egress-gateway-tls-origination/index.md

```diff
@@ -242,15 +184,14 @@ be done by the egress gateway, as opposed to by the sidecar in the previous exam
 1.  Send an HTTP request to [http://edition.cnn.com/politics](https://edition.cnn.com/politics).
 
     {{< text bash >}}
-    $ kubectl exec -it $SOURCE_POD -c sleep -- curl -sL -o /dev/null -D - http://edition.cnn.com/politics
+    $ kubectl exec -it $(kubectl get pod -l app=sleep -o jsonpath={.items..metadata.name}) -c sleep -- curl -sL -o /dev/null -D - http://edition.cnn.com/politics
     HTTP/1.1 200 OK
     ...
     content-length: 150793
     ...
     {{< /text >}}
 
-    The output should be the same as in the [TLS Origination for Egress Traffic](/docs/tasks/traffic-management/egress/egress-tls-origination/)
-    example, with TLS origination: without the _301 Moved Permanently_ message.
+    The output should be without the _301 Moved Permanently_ message.
```

Comment:
I think maybe this text would be clearer: "The output should not contain a _301 Moved Permanently_ message."?

Reply:
Fixed in https://github.com/istio/istio.io/pull/6194/commits/ceb7542d4b1d6d2109d1f29b9fd420f6e6159a9d.